openwisp_users/tests/test_admin.py

```diff
@@ -507,9 +542,85 @@ def test_admin_add_user_org_required(self):
         self.assertEqual(queryset.count(), 0)
         self.assertContains(res, 'errors field-organization')
 
+    def test_admin_user_add_form(self):
+        self.client.force_login(self._get_admin())
+        r = self.client.get(reverse(f'admin:{self.app_label}_user_add'))
+        self.assertContains(r, 'first_name')
+        self.assertContains(r, 'last_name')
+        self.assertContains(r, 'phone_number')
+        self.assertContains(r, 'groups')
+
+    def test_add_user_without_group(self):
+        admin = self._create_admin()
+        self.client.force_login(admin)
+        params = dict(
+            username='testadd',
+            email='test@testadd.com',
+            password1='tester',
+            password2='tester',
+            is_staff=True,
+            is_superuser=True,
+        )
+        params.update(self.add_user_inline_params)
+        params.update(self._additional_params_add())
+        params.update(
+            {
+                f'{self.app_label}_organizationuser-TOTAL_FORMS': 1,
+                f'{self.app_label}_organizationuser-INITIAL_FORMS': 0,
+                f'{self.app_label}_organizationuser-MIN_NUM_FORMS': 0,
+                f'{self.app_label}_organizationuser-MAX_NUM_FORMS': 1,
+            }
+        )
+        res = self.client.post(
+            reverse(f'admin:{self.app_label}_user_add'), params, follow=True
+        )
+        self.assertEqual(res.status_code, 200)
+        self.assertContains(
+            res, 'A staff user must belong to a group, please select one.'
+        )
+
+    def test_add_user_with_group(self):
```

Comment:
instead of testing this, can you please test a non staff and non superuser?
We expect the user to be created normally in this case